packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1837,7 +1837,9 @@ class TextInput {
         final Map<String, dynamic> firstArg = args[1] as Map<String, dynamic>;
         _currentConnection!._client.performPrivateCommand(
           firstArg['action'] as String,
-          firstArg['data'] as Map<String, dynamic>,
+          firstArg['data'] == null
```

Comment:
nit: `as Map<String, dynamic>?`

Reply:
Oh duh, that's awesome that we can do that 👍